src/renderers/dom.js

```diff
@@ -162,6 +166,24 @@ const getDefaultDomOptions = () : DomOptions => {
     return {};
 };
 
+function addXmlNamespace(node : ElementNode | TextNode | ComponentNode<*>, xmlns : string = 'http://www.w3.org/2000/svg') {
+    if (node.type === NODE_TYPE.TEXT) {
+        return;
+    }
+    let ns = xmlns;
+    if (typeof node.props.xmlns === 'string') {
+        ns = node.props.xmlns;
+    } else {
+        node.props.xmlns = ns;
```

Comment:
When would we expect the caller to not explicitly pass `xmlns` for an SVG? Do we need this line?